packages/xo-server-netbox/src/index.js

```diff
@@ -699,6 +699,25 @@ class Netbox {
     // Use the first IPs found in vm.addresses as the VMs' primary IPs in
     // Netbox, both for IPv4 and IPv6
 
+    const getPrimaryIps = addresses => {
+      const primaryIps = {}
+      const keys = Object.keys(addresses).sort()
```

Comment:
Remove the sort on this, as it will break for `10` vs `2`.

The default order should be good.